bee-tangle/tests/helper.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 IOTA Stiftung
```

Comment:
Remove

Reply:
For me it's still there. Maybe it's on my end. Dunno.